test/e2e/scenario_runtime-http_test.go

```diff
@@ -18,6 +18,7 @@ import (
 var runtimeSupportMap = map[string][]string{
 	"node":       {"pack", "s2i"},
 	"go":         {"pack"},
+	"rust":       {"pack"},
```

Comment:
@jrangelramos do you recall why this has not been enabled before?

Reply:
I think it was just never added since scratch of tests.